test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpLogExporterTests.cs

```diff
@@ -458,5 +460,48 @@ public void CheckToOtlpLogRecordExceptionAttributes()
             Assert.Contains(SemanticConventions.AttributeExceptionStacktrace, otlpLogRecordAttributes);
             Assert.Contains(logRecord.Exception.ToInvariantString(), otlpLogRecordAttributes);
         }
+
+        [Fact]
+        public void CheckToOtlpLogRecordRespectsAttributeLimits()
+        {
+            var sdkLimitOptions = new SdkLimitOptions
+            {
+                AttributeCountLimit = 3, // 3 => LogCategory, exception.type and exception.message
+                AttributeValueLengthLimit = 8,
+            };
+
+            var logRecords = new List<LogRecord>();
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.AddInMemoryExporter(logRecords);
+                });
+            });
+
+            var logger = loggerFactory.CreateLogger("OtlpLogExporterTests");
+            logger.LogInformation(new Exception("I'm the exception message."), "Exception Occurred");
+
+            var logRecord = logRecords[0];
+            var otlpLogRecord = logRecord.ToOtlpLog(sdkLimitOptions);
+
+            Assert.NotNull(otlpLogRecord);
+            Assert.True(otlpLogRecord.DroppedAttributesCount > 0, "Attributes dropped count is unset");
```

Comment:
I see other tests checking the `DroppedAttributesCount` (in the same way?) like this
```suggestion
            Assert.Equal(1u, otlpLogRecord.DroppedAttributesCount);
```

Reply:
I'm be honest, I'm not that happy with that change, but I change it 👍.

That change means, we are always required to change this test when we add new attributes to the LogRecordExtensions function, which increases the maintain efforts a little bit.
Of cause we are required to change that test, when we add/change attributes that are added before the exception attributes, but not after.